ci/workflows/pull-request.yml

```diff
@@ -656,21 +656,6 @@ steps:
     cancel_on_build_failing: true
     retry: *auto-retry
 
-  - label: "misc check"
-    command: "ci/scripts/misc-check.sh"
-    if: |
-      !(build.pull_request.labels includes "ci/pr/run-selected") && build.env("CI_STEPS") == null
-      || build.pull_request.labels includes "ci/run-misc-check"
-      || build.env("CI_STEPS") =~ /(^|,)misc-check(,|$$)/
-    plugins:
-      - docker-compose#v5.5.0:
-          run: rw-build-env
-          config: ci/docker-compose.yml
-      - shellcheck#v1.2.0:
-          files: ./**/*.sh
```

Comment:
shellcheck is removed. We didn't set `recursive-blob`, so it didn't check any shell scripts! Now there are too many lint errors, and I think it's not worth fixing